utils/tx.ts

```diff
@@ -332,3 +350,9 @@ export async function getTreasuryBurn(currencyId : BN){
   const treasuryBalance = await api.query.xyk.treasuryBurn(currencyId);
   return treasuryBalance.toHuman();
 }
+
+export async function getLock(accountAddress:string, assetId : BN){
+  const api = getApi();
+  const locksResponse = await api.query.tokens.locks(accountAddress,assetId);
+  return locksResponse.toHuman() as BalanceLock[];
```

Comment:
Not sure why this is required. When you use .toHuman() does it not auto format it to a JSON?
https://polkadot.js.org/docs/api/start/types.basics#everything-is-a-type
Also, do we need to always return JSON? or can we not just work with the object itself and use it's getters such as given in:
https://polkadot.js.org/docs/api/start/types.basics#working-with-structures

Reply:
When .ToHuman() It creates an AnyJSON object, which is good. But the issue is that when the object is a list of some other objects, since the type is not defined, you can not access to toHuman()[0]  [to access to the first property].
For that and some other reasons, I would like to start using responses with types. 

```
Also, do we need to always return JSON? or can we not just work with the object itself and use it's getters such as given in:
```
I'll look at the getters and be back to you ! Maybe we can avoid casting types everytime and use those.

Reply:
done a trick converting JSON. let me know if that fits ur expectations :)